check50.py

```diff
@@ -94,7 +94,7 @@ def main():
                 # Terminate if no response.
                 if pings > 45:
                     cprint("check50 is taking longer than normal!", "red", file=sys.stderr)
-                    cprint("more info at: https://cs50.me/check50/results/{}/{}".format(username, commit_hash), "red", file=sys.stderr)
+                    cprint("more info at: https://cs50.me/checks/{}".format(commit_hash), "red", file=sys.stderr)
```

Comment:
For consistency with v1, can we just do `See https://cs50.me/checks/{} for more detail.`?